### DISCUSSION TITLE
Caddy with Cloudflare plugin and ddns

### DISCUSSION THREAD

**Discussion by mr-n0cturne** [tteck/Proxmox#3486] [Help Needed]
Hello,

how can I use cloudflare and ddns with caddy lxc? I already have a prepared Caddyfile. 

Cheers

**Comment by belgriffinite** [comment]
I was going to ask the same thing but then decided to search first, and saw your question. I am wondering the same thing. I love these scripts, but caddy really needs custom modules for almost every install. Hmm
